modules/images-configuration-file.adoc

```diff
@@ -60,21 +51,18 @@ spec:
 status:
   internalRegistryHostname: image-registry.openshift-image-registry.svc:5000
 ----
-<1> `Image`: Holds cluster-wide information about how to handle images. The canonical, and only valid name is `cluster`.
```

Comment:
I like that you removed the callouts here because the table above highlights all of this information. Do you think that all callouts (e.g., here: https://102644--ocpdocs-pr.netlify.app/openshift-enterprise/latest/openshift_images/image-configuration.html#images-configuration-allowed_image-configuration) could be removed in these examples, since the table is listed above? You might have a reason for not doing that. And if you don't want to do it, you could add it to the spreadsheet for a future consideration.

Reply:
Ah, I didn't because this module is used in another assembly and it didn't look like the entire table belonged there. I'll take a deeper look after break.